lib/AST/USRGeneration.cpp

```diff
@@ -249,6 +258,17 @@ swift::USRGenerationRequest::evaluate(Evaluator &evaluator, const ValueDecl* D)
   return NewMangler.mangleDeclAsUSR(D, getUSRSpacePrefix());
 }
 
+bool ide::printModuleUSR(ModuleEntity Mod, raw_ostream &OS) {
+  if (auto *D = Mod.getAsSwiftModule()) {
+    StringRef moduleName = D->getName().str();
+    return clang::index::generateFullUSRForTopLevelModuleName(moduleName, OS);
+  } else if (auto ClangM = Mod.getAsClangModule()) {
+    return clang::index::generateFullUSRForModule(ClangM, OS);
+  } else {
+    return true;
+  }
+}
```

Comment:
The logic inside `evaluate` isn't used for this feature, but I left it there as well for completion's sake. It was indeed duplicated so I'm making it call the original `printModuleUSR`, but I could also remove it entirely since it's not relevant for the feature.

Reply:
I'd recommend to remove if it's not used, generally it's better to not leave dead code in, if possible.

Reply:
Fair enough, I've also removed the other changes that ended up being not necessary anymore.